cypress/e2e/course/addBBBToolToCourseInNBC.feature

```diff
@@ -68,7 +68,7 @@ Feature: Teacher can add and remove BBB tool in the course in NBC with admin per
 		# Admin disables the video conference option in the school settings page and teacher can not add the bbb tool in the course
 		Given I am logged in as a '<admin>' at '<namespace>'
 		When I click on administration in menu
-		When I click on manage school card
+    When I go to school administration
```

Comment:
This step is refactored in my PR now:
for NBC:
- When I click on administration in menu
- When I click on sub menu school

for BRB/dBC:
-When I click on administration in menu
-When I click on sub menu school
-When I go to new school administration page